src/redux/initialState.js

```diff
@@ -1351,6 +1351,13 @@ const initialState = {
     // { id: 'Brand9', name: 'Brand9', image: 'https://i.postimg.cc/HLPHqTFm/17.jpg' },
     // { id: 'Brand10', name: 'Brand10', image: 'https://i.postimg.cc/C5bgnk5M/18.jpg' },
   ],
+  productCategories: [
+    { id: 'furniture', name: 'Furniture' },
+    { id: 'chair', name: 'Chair' },
+    { id: 'table', name: 'Table' },
+    { id: 'sofa', name: 'Sofa' },
+    { id: 'bedroom', name: 'Bedroom' },
```

Comment:
Jeżeli brakuje jakiegoś produktu w danej kategori to uzupełnij proszę listę produktów o produkty z danej kategorii